### PR TITLE
Fix/minor comments review sarah

### LINKED ISSUE #449: Fix minor comments from Carpentries Lab review
Look at https://github.com/carpentries-lab/reviews/issues/25, see if there are any minor comments that we missed and can still improve.
```[tasklist]
### Tasks
- [x] listed browser support is not matched to jupyter browser support notably, Windows' current default browser is a chromium browser and therefore supported (and on general web compatibility, better than safari by a nontrivial margin) this should be updated to include more windows users without having to install
- [x] a glossary of key terms or links out to definitions in an external glossary e.g. Glosario.
```

### PR DESCRIPTION
**Pull request by carschno**
- Update list of supported browsers
- Add links to Wikipedia/Glossary
- Minor language fixes
- Update software version and installation check

Closes #449

### CODE REVIEW

File: episodes/1-introduction.Rmd [hunk 1]
```diff
@@ -57,15 +57,17 @@ A neural network consists of connected computational units called **neurons**. E
 - most of the time, each neuron conducts 3 main operations:
   + take the weighted sum of the inputs where ($w_1, w_2, ... $) indicate weights
   + add an extra constant weight (i.e. a bias term) to this weighted sum
-  + apply a non-linear function to the output so far (using a predefined activation function such as the ReLU function)
+  + apply a non-linear function to the output so far (using a predefined activation function such as the [ReLU](https://en.wikipedia.org/wiki/Rectifier_(neural_networks)) function)
```
Comment (svenvanderburg) [inline episodes/1-introduction.Rmd:60]: This will be changed again with #498 , but approving for now.

File: episodes/1-introduction.Rmd [hunk 2]
```diff
@@ -437,16 +443,17 @@ Keras also benefits from a very good set of [online documentation](https://keras
 Follow the instructions in the [setup]({{ page.root }}//setup) document to install Keras, Seaborn and Sklearn.
 
 ## Testing Keras Installation
-Lets check you have a suitable version of tensorflow installed.
+Keras is available as a module within TensorFlow, as described in the [setup]({{ page.root }}//setup).
+Let's therefore check you have a suitable version of TensorFlow installed.
```
Comment (svenvanderburg) [inline episodes/1-introduction.Rmd:447]:
```suggestion
Let's therefore check whether you have a suitable version of TensorFlow installed.
```

**Review by svenvanderburg — APPROVED** [review]
Looks good, I left a few minor comments.

File: episodes/1-introduction.Rmd (hunk 1 above)
Comment (carschno) [reply episodes/1-introduction.Rmd]: Indeed, the link is unnecessary now. Merged.

**Pull request merged**